src/cli/cluster/worker.js

```diff
@@ -82,8 +85,15 @@ module.exports = class Worker extends EventEmitter {
   }
 
   onMessage(msg) {
```

Comment:
What do you think about changing this to `onMessage([type, message])` rather than accessing the value via numeric index within the function itself?  It's a little bit more expressive, which can help people grok it at a glance.  To be honest, I'm not even sure if `type` and `message` are appropriate because I'm not really sure what is even stored in the `msg` array, which sort of just underscores the need for more expressiveness.